projects/admin/src/app/circulation/patron/loan/loan.component.ts

```diff
@@ -17,19 +17,33 @@
 
 import { Component, OnDestroy, OnInit } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
+import { DateTranslatePipe } from '@rero/ng-core';
 import { User, UserService } from '@rero/shared';
+import { BsModalRef, BsModalService } from 'ngx-bootstrap/modal';
 import { ToastrService } from 'ngx-toastr';
+import { Item, ItemAction, ItemNoteType, ItemStatus } from 'projects/admin/src/app/class/items';
+import { ItemsService } from 'projects/admin/src/app/service/items.service';
+import { PatronService } from 'projects/admin/src/app/service/patron.service';
 import { forkJoin, Subscription } from 'rxjs';
-import { Item, ItemAction, ItemNoteType, ItemStatus } from '../../../class/items';
-import { ItemsService } from '../../../service/items.service';
-import { PatronService } from '../../../service/patron.service';
+import { FixedDateFormComponent } from './fixed-date-form/fixed-date-form.component';
+
+
+/** Interface to declare a special circulation settings */
+export interface CirculationSetting {
+  key: string;    /** the setting internal key */
+  label: string;  /** the setting label to display to user */
+  value: any;     /** the setting value */
+}
+
 
 @Component({
   selector: 'admin-loan',
-  templateUrl: './loan.component.html'
+  templateUrl: './loan.component.html',
+  providers: [ DateTranslatePipe ]
 })
 export class LoanComponent implements OnInit, OnDestroy {
 
+  // COMPONENT ATTRIBUTES ============================================
```

Comment:
cosmetic: if this comment is a new best practice, it will be great to have that everywhere is needed

Reply:
@lauren-d 
This isn't yet a "best practice". This is a trick for me to more easily find/class part of a class when a component become very long.